docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,162 @@ described below, and relative to the joint TMREL.
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+
+Vivarium modelling strategy
++++++++++++++++++++++++++++
+
+First, we convert the GBD 500g-2weeks birthweight-ga bins/categories to a joint continuous distribution using Abie's notebook (add link?). We assume a uniform distribution within each bin/category. 
+
+.. note ::
+    That this is likely biasing towards overestimating extreme birthweights or gestational ages. For example, in the 0-500g category, most babies are probably pretty close to 500g, not equally probable to be 1 gram versus 499 grams.
+
+Because the relative risks from GBD are for all-cause mortality in the early and late neonatal period, we first define all-cause mortality rate (ACMR) as the sum of: 
+
+   - mortality from causes that are affected by lbwsg and modelled in the sim (green)
+   - mortality from causes that are affected by lbwsg but not modelled in the sim (blue)
+   - mortality from causes that are unaffected by lbwsg, modelled or not modelled (salmon)
+
+An example of these causes from the large-scale-food fortification concept model diagram is shown below: 
+
+.. image:: causes_equation.svg
```

Comment:
I like how this table lets you visualize what's going on.  Nice job!

Reply:
ya...maybe we can change the aesthetics of it later ....